release-notes/v1_39.md

```diff
@@ -65,6 +65,16 @@ With the introduction of a new variable `${defaultBuildTask}` it is now possible
 
 TODO@isidorn https://github.com/microsoft/vscode/issues/68084
 
+## Contributions to extensions
+
+### Java 13 support 
+
+[Java 13](http://jdk.java.net/13/) is out and VS Code is ready for it. It supports Java 13 through latest [Java Extension](https://marketplace.visualstudio.com/items?itemName=redhat.java). For developers use Java 12 with preview features, you will need to migrate to 13 to keep working with them.
+
+### Deprecation mark for deprecated Java class, method or field
+
+The Java language server adopts the [Deprecated Diagnostics Tag](/updates/v1_37#_diagnosticstagdeprecated) which renders the code in problem with a strike-through line. 
```

Comment:
... which renders the code that references deprecated types/members with a strike-through...